terraform/helm/aptos-node/files/haproxy.cfg

```diff
@@ -67,6 +67,7 @@ backend validator-api
 {{- end }}
 
 {{- range $index, $config := $.Values.fullnode.groups }}
+{{- if lt $.Values.i (int $.Values.numFullnodes) }}
```

Comment:
How do we change the load test to start using the FNs? Is there a flag that we can set?

Reply:
Yeah simply specify num_fullnodes below num_validators in the testnet config